app/javascript/components/visualization/controls/SpatialSelector.jsx

```diff
@@ -9,16 +9,75 @@ function getSpatialOptions(allSpatialGroups) {
   return clusterList.map(group => {return { label: group.name, value: group.name }})
 }
 
-/** component for displaying a spatial group selector
+/** Small "x" button to clear a message */
+function InlineClearButton({ setShowMessage, issueType }) {
+  return (
+    <button
+      className={`clear-${issueType}-inline`}
+      onClick={() => setShowMessage(false)}
+      title="Dismiss message"
+    >
+      <svg height="14" width="14" viewBox="0 0 20 20">
```

Comment:
Just a question - is there a reason for using this SVG rather than the normal Font Awesome X icon?

Reply:
This "x" exactly matches the "x" in the select options, which seemed consistent, visually tidy, and easy.